CoreConversationView/test/unit/src/au/gov/asd/tac/constellation/views/conversationview/ConversationViewTopComponentNGTest.java

```diff
@@ -42,10 +45,26 @@ public class ConversationViewTopComponentNGTest {
 
     @BeforeClass
     public static void setUpClass() throws Exception {
+        if (!FxToolkit.isFXApplicationThreadRunning()) {
+            FxToolkit.registerPrimaryStage();
+        }
     }
 
     @AfterClass
     public static void tearDownClass() throws Exception {
+        try {
+            FxToolkit.cleanupStages();
+        } catch (final TimeoutException ex) {
+            LOGGER.log(Level.WARNING, "FxToolkit timedout trying to cleanup stages", ex);
+        } catch (Exception e) {
+            if (e.toString().contains("HeadlessException")) {
+                System.out.println("\n**** EXPECTED TEARDOWN ERROR: " + e.toString());
+            } else {
+                System.out.println("\n**** UN-EXPECTED TEARDOWN ERROR: " + e.toString());
+                throw e;
+            }
```

Comment:
More for curiosity: is there a reason why need to include this extra condition in this case? Noting that we haven't needed to include this elsewhere so wondering what happens here that needs this extra catch?